packages/cursorless-engine/src/processTargets/targets/SubTokenWordTarget.ts

```diff
@@ -43,11 +43,12 @@ export default class SubTokenWordTarget extends BaseTarget {
     return getDelimitedSequenceRemovalRange(this);
   }
 
-  protected getCloneParameters() {
+  protected getCloneParameters(): SubTokenTargetParameters {
     return {
       ...this.state,
       leadingDelimiterRange: this.leadingDelimiterRange_,
       trailingDelimiterRange: this.trailingDelimiterRange_,
+      insertionDelimiter: this.insertionDelimiter,
```

Comment:
All this ceremony wasn't for nothing, it seems! I added the return type here for autocomplete.

Reply:
So you don't get autocomplete for free from the generic at the top?